shared/AppInsightsCore/src/JavaScriptSDK.Interfaces/IProcessTelemetryContext.ts

```diff
@@ -73,4 +86,9 @@ export interface IProcessTelemetryContext {
      *                  order then the next plugin will be NOT set.
      */
     createNew: (plugins?:IPlugin[]|ITelemetryPluginChain, startAt?:IPlugin) => IProcessTelemetryContext;
+
+    /**
+     * Set the function to call when the current chain has executed all processNext or unloadNext items.
+     */
+    onComplete: (onComplete: () => void) => void;
```

Comment:
Adding a "callback" that will be called once all of the plugins have been processed.